### PR TITLE
Farm 01 form login

### PR DESCRIPTION
**Pull request by CaroD59**
Front FormLogin finished for the moment

### CODE REVIEW

**Review by Erriep — DISMISSED** [review]

**Review by Erriep — DISMISSED** [review]

File: src/components/Login/index.jsx [hunk 1]
```diff
@@ -1,4 +1,53 @@
...
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import LoginForm from './LoginForm';
+
+export default function Login() {
+  const Background = styled.div`
+    background-image: url(./Images/welcomefont.png);
+    background-size: cover;
+    height: 100vh;
+  `;
```
Comment (LoicBrassart) [inline src/components/Login/index.jsx]: This sould be in a `style.jsx` file

File: src/components/Login/index.jsx [hunk 2]
```diff
@@ -1,4 +1,53 @@
...
+  const adminUser = {
+    email: 'dorchies.c@gmail.com',
+    password: 'lol123',
+  };
+
+  const [user, setUser] = useState({ email: '' });
+  const [error, setError] = useState('');
+
+  const Loggedin = (details) => {
+    console.log(details);
```
Comment (LoicBrassart) [inline src/components/Login/index.jsx]: No `console.log` !

File: src/components/Login/index.jsx [hunk 3]
```diff
@@ -1,4 +1,53 @@
...
+    if (
+      details.email === adminUser.email &&
+      details.password === adminUser.password
+    ) {
+      console.log('Logged in o/');
```
Comment (LoicBrassart) [inline src/components/Login/index.jsx]: Same!

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some minor Evils

**Review by Erriep — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
NmE!

**Pull request merged**